SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp

```diff
@@ -36,6 +36,7 @@ USpatialGDKSettings::USpatialGDKSettings(const FObjectInitializer& ObjectInitial
 	, bBatchSpatialPositionUpdates(false)
 	, MaxDynamicallyAttachedSubobjectsPerClass(3)
 	, bEnableServerQBI(true)
+	, bEnableClientResultTypes(true)
```

Comment:
```suggestion
	, bEnableClientResultTypes(false)
```

Reply:
letting CI run with the flag flip, then will flip it back before merging just to sanity check

Reply:
flipped, all passed with and without